frontend/src/components/University/MainUniversitySidebar.jsx

```
@@ -18,30 +19,33 @@
-                navigate(`/university/${uni.name}`)}}
+    <div className="w-full h-screen overflow-y-auto border-r bg-white p-4 shadow-md">
+  <h1 className="text-xl font-bold mb-6 text-gray-800">Explore Universities</h1>
+  {universities.map((uni) => (
+    <div
+      key={uni.id}
+      className="mb-6 bg-gray-50 rounded-2xl shadow-sm p-4 hover:shadow-md transition-shadow"
+    >
+      <h2
+        className="flex items-center gap-2 text-blue-700 text-lg font-semibold cursor-pointer hover:underline"
+        onClick={() => navigate(`${uni.name}`)}
+      >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)